### ISSUE TITLE
Some form elements are missing labels

### ISSUE THREAD

**Issue by bedmison** [OpenDSA/OpenDSA-LTI#192]
All form elements should have labels, but some do not. For example, the "title" field in https://opendsa-server.cs.vt.edu/configurations/book is missing a label.

To solve this problem, you need to fix at least (1) of the following:

- Element does not have an implicit (wrapped) <label>
- Element does not have an explicit <label>
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element has no placeholder attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by bedmison** [comment]
Deferred until Rails 8 upgrade feasibility is determined.

**Comment by cashaffer** [comment]
In particular, the issue has to to with the viability of Formstastic. If we have to replace that with a new forms system, then all these forms will get replaced.
